src/api/ai_help.rs

```diff
@@ -503,15 +506,13 @@ pub async fn ai_help(
                                     message_id,
                                     parent_id,
                                     created_at: Some(created_at.naive_utc()),
-                                    search_duration: default_meta_big_int(
-                                        ai_help_req_meta.search_duration.map(|d| d.as_millis()),
-                                    ),
-                                    response_duration: default_meta_big_int(Some(
-                                        response_duration.as_millis(),
-                                    )),
+                                    embedding_duration: default_meta_duration(ai_help_req_meta.embedding_duration),
+                                    search_duration: default_meta_duration(ai_help_req_meta.search_duration),
+                                    response_duration: default_meta_duration(Some(response_duration)),
                                     query_len: default_meta_big_int(ai_help_req_meta.query_len),
                                     context_len: default_meta_big_int(ai_help_req_meta.context_len),
                                     response_len: default_meta_big_int(Some(context.len)),
+                                    embedding_model: ai_help_req_meta.embedding_model.unwrap_or(""),
```

Comment:
nit: `unwrap_or_default()` looks a bit more idiomatic
```suggestion
                                    embedding_model: ai_help_req_meta.embedding_model.unwrap_or_default(),
```